app/src/androidTest/java/com/m2049r/xmrwallet/util/HelperTest.java

```diff
@@ -0,0 +1,30 @@
+package com.m2049r.xmrwallet.util;
+
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.SmallTest;
+
+import com.m2049r.xmrwallet.model.Wallet;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Created by bruno on 14/11/2017.
+ */
+
+@RunWith(AndroidJUnit4.class)
+@SmallTest
+public class HelperTest {
+
+    /**
+     *
+     */
+    @Test
+    public void checkValidAddressExample_1() {
+        // random generated address by https://xmr.llcoins.net/addresstests.html
+        String publicAddr = "4GKacjHuTGeReA7PvxPhehDswu3RwpnAqeJV2kdU3MLMNUr4Lt9pxcRiWU2E6BoDReADhCPtHfvNFh1YVNLFoZE5YtaSMc8MSbGGudRftK";
+        Assert.assertTrue(Wallet.isAddressValid(publicAddr, false));
```

Comment:
Why is this test part of `HelperTest`? Shouldn't it be in something like `WalletTest`? In this file I would expect tests that cover the methods of the `Helper` class.

Reply:
It's a simple dummy test, just an example part of the integration for the tests. To be fair Wallet is mostly JNI mapping to the actual native wallet lib. I think it already has it's own tests in the monero tree. We could / should mock these in the future and focus the tests on monerujo's scope. 

I'm already writing my tests for my project & fork. I can delay submitting all this later if it's something we want on this project

Reply:
If you write tests and submit them when you are done that would be amazing.